### PR TITLE
Compare

### PR DESCRIPTION
**Pull request by TheDiren**
Added clean weight function. Working but there might be still some issues when there are multiple inspections in close distance.
Also add a compare data function, to request mean of multiple hives, but that was more for trying out.
There might be some additions to seeder, etc. which actually are not helpful.
also added some tips to readme.

### CODE REVIEW

**Review by pvgennip — CHANGES_REQUESTED** [review]
This should not be in the readme, since it is branch specific and will later on be merged in other branches

File: app/Measurement.php [hunk 1]
```diff
@@ -126,7 +126,7 @@ public static function getIdByAbbreviation($abbreviation)
 
     public static function getMatchingMeasurements()
     {
-        return ['bv','w_v','weight_kg','t_i','t_0','t_1','s_bin_71_122','s_bin_122_173','s_bin_173_224','s_bin_224_276','s_bin_276_327','s_bin_327_378','s_bin_378_429','s_bin_429_480','s_bin_480_532','s_bin_532_583','s_bin_0_201','s_bin_201_402','s_bin_402_602','s_bin_602_803','s_bin_803_1004','s_bin_1004_1205','s_bin_1205_1406','s_bin_1406_1607','s_bin_1607_1807','s_bin_1807_2008'];
+        return ['bv','w_v','weight_kg', 'mean_weight_kg', 'weight_kg_noOutlier', 't_i','t_0','t_1','s_bin_71_122','s_bin_122_173','s_bin_173_224','s_bin_224_276','s_bin_276_327','s_bin_327_378','s_bin_378_429','s_bin_429_480','s_bin_480_532','s_bin_532_583','s_bin_0_201','s_bin_201_402','s_bin_402_602','s_bin_602_803','s_bin_803_1004','s_bin_1004_1205','s_bin_1205_1406','s_bin_1406_1607','s_bin_1607_1807','s_bin_1807_2008'];
```
Comment (pvgennip) [inline app/Measurement.php:129]: Please remove these, since they will never be present in Flashlog data

File: routes/api.php
Comment (pvgennip) [inline routes/api.php:1]: Maybe include in default way of getting data, does not have to be a separate function

**Review by pvgennip — APPROVED** [review]

**Comment by TheDiren** [comment]
I removed changes in the old app and added the default way of getting data to trello

**Pull request merged**